pydantic/networks.py

```diff
@@ -131,6 +131,8 @@ def __hash__(self) -> int:
 """A type that will accept any MongoDB DSN."""
 KafkaDsn = Annotated[Url, UrlConstraints(allowed_schemes=['kafka'], default_host='localhost', default_port=9092)]
 """A type that will accept any Kafka DSN."""
+NatsDsn = Annotated[MultiHostUrl, UrlConstraints(allowed_schemes=['nats', 'tls', 'ws'], default_host='localhost', default_port=4222)]
+"""A type that will accept any NATS DSN."""
```

Comment:
we need to add better documentation for all these types, but while you're adding this, please could you expand the documentation a bit here.

In particular a link to NATS and some context on what it is would be useful.

I for one have no idea what "NATS" is in this context.

Reply:
Ok. I agree it would be better.